lib/pipeline.py

```diff
@@ -261,32 +260,37 @@ def generate_fasta(self, chunks):
             tsv_chunk = "%d-%d" % (split_start, split_end)
             fasta_chunk = "%d-%d" % (split_start*2-1, split_end*2)
             generate_fasta_params = {
-                'peptide_sequence_length'   : self.peptide_sequence_length,
                 'downstream_sequence_length': self.downstream_sequence_length,
                 'proximal_variants_file'    : self.proximal_variants_file,
             }
-            split_fasta_file_path = "%s_%s" % (self.split_fasta_basename(), fasta_chunk)
-            if os.path.exists(split_fasta_file_path):
-                status_message("Split FASTA file for Entries %s already exists. Skipping." % (fasta_chunk))
-                continue
             if self.input_file_type == 'pvacvector_input_fasta':
+                split_fasta_file_path = os.path.join(self.tmp_dir, "{}.fa.split_{}".format(self.sample_name, fasta_chunk))
```

Comment:
This version ended up copy+pasted a few times in this file.  If it was worth making a `split_fasta_basename()` to begin with, is it also worthwhile to have a version that works for this path (either via an option or a separate `pvacvector_split_fasta_basename()` or the like?

Reply:
good idea. I made that change.